### DISCUSSION TITLE
Falha ao atualizar EPG

### DISCUSSION THREAD

**Discussion by pedromdm** [LITUATUI/M3UPT#84] [General]
Olá,

Sou recente no uso da IPTV porque mudei para um serviço que apenas me disponibiliza internet. 
Estou à procura de um player, mas todos os que tenho experimentado (televizo,tivimate, m3u iptv, IPTV...) não estão a carregar os EPGs. Já tentei mudar para a extensão .gz mas tenho o mesmo problema.

Podem ajudar?

Obrigado

**Comment by LITUATUI** [comment]
Experimente um dos [leitores de IPTV recomendados](https://m3upt.com/#-leitores-de-iptv-recomendados).

**Reply by pedromdm** [reply]
Obrigado, instalei o OTT TV e este carrega os EPG.  Estou agora com outro problema. Tenho uma boa NET e os canais estão sempre a parar. Alguma solução para isto? não me parece problemas de conectividade. Estou a usar um XIAOMI TV STICK...

**Reply by LITUATUI** [reply]
Usaste o código de backup para a OTT Navigator?
